core/app/models/spree/variant.rb

```diff
@@ -211,7 +211,7 @@ def deleted?
     # Assign given options hash to option values.
     #
     # @param options [Array] array of hashes with a name and value.
```

Comment:
The doc is also wrong, as it should be `[Hash]`. Nice if we fix it now.

Reply:
I think the brackets only indicate that this is the type part of the line, not that it's an array. I can be specific and do `[Array<Hash>]`.

Reply:
Oh, yeah, sorry, I was confused. Thanks for your change.